controllers/article-controller.js

```diff
@@ -12,7 +12,22 @@ exports.getArticleById = (req, res, next) => {
 }
 
 exports.getAllArticles = (req, res, next) => {
-    fetchAllArticles()
+    let { sort_by, order } = req.query;
+
+    const validSortColumns = ['author', 'title', 'article_id', 'topic', 'created_at', 'votes', 'comment_count']
+    const validOrders = [ 'asc', 'desc']
+        
+    if (sort_by && !validSortColumns.includes(sort_by)){
+      return next({ status: 400, msg: 'Bad Request: Invalid Sort Column'});
+    }
+    if (order && !validOrders.includes(order.toLowerCase())){
+      return next({ status: 400, msg: 'Bad Request: Invalid Sort Order'})
+    }
```

Comment:
I'd suggest moving this to your model so it is consistent with your other error handling, and using Promise.reject to pass it to your controller catch block.